Prace_domowe/Praca_domowa2/Marcelina_Kurek/PD2.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>Bardzo dobry pomysł żeby to przedstawić na wykresie. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>